### PR TITLE
Feat/captcha

### PR DESCRIPTION
**Pull request by jona159**

### CODE REVIEW

**Review by YouQam — APPROVED** [review]
LGTM
`Note`: don't merge until approve/merge  [Feat/captcha #282 ](https://github.com/mint-o-badges/badgr-ui/pull/282/commits/0d92206773190b6c5cd739610000ea9466ee170f)

File: apps/badgeuser/api.py
Comment (jona159) [reply apps/badgeuser/api.py:113]: commented out email_domain as well

File: apps/badgeuser/api.py
Comment (jona159) [reply apps/badgeuser/api.py:120]: deleted

File: apps/badgeuser/api.py
Comment (jona159) [reply apps/badgeuser/api.py]: I commented out the whole spamfilter for now, this topic will be discussed in a seperate meeting

File: apps/badgeuser/api.py
Comment (jona159) [reply apps/badgeuser/api.py]: need to wait for the results from abovementioned meeting

File: .docker/etc/settings_local.prod.py.example
Comment (jona159) [reply .docker/etc/settings_local.prod.py.example:150]: Done

**Comment by timber-they** [comment]
@jona159 I approved the changes but now you need to resolve merge conflicts

**Pull request merged**
